app/views/shared/_messages.html.slim

```diff
@@ -2,4 +2,4 @@
   - flash.each do |name, msg|
     - if msg.present? && msg.is_a?(String)
       div class="alert alert-#{name}" role='alert'
-        = msg
+        = sanitize(msg)
```

Comment:
Or, as Zach mentioned, remove the formatting. I don't we have formatting in any other flash message, right? Why does "phone of record" need to be in bold?